NGitLab/Models/PackageStatus.cs

```diff
@@ -0,0 +1,11 @@
+namespace NGitLab.Models
+{
+    public enum PackageStatus
```

Comment:
Use C# name (e.g. `Default`) and the `EnumMember(Value = "")]` attribute to map the actual name

Reply:
Happy to make this change, but I am curious what your suggestion is when it comes to things like this:

https://github.com/ubisoft/NGitLab/pull/524/files#diff-27c124206cca88b65b78a9a2fe44d27dfcc3fab2b510dc692c6507c5cc7d8833R50

```
url = Utils.AddParameter(url, "status", query.Status);
```

i.e. when using the Enum value to construct the URL that is requested.  With the change that you have suggested in play, the constructed URL doesn't work, as the GitLab server returns an error...

<img width="1440" alt="image" src="https://github.com/ubisoft/NGitLab/assets/1271146/9707bbe2-d48e-457c-bfee-f0506be1c888">

Any suggestions?

Reply:
@meziantou just wanted to follow up on this one.  Any ideas on the best course of action here? Thanks!

Reply:
Sorry for the delay. I think the best solution is to improve `Utils.AddParameter` to better support enums:

```
public static string AddParameter<T>(string url, string parameterName, T value)
{
    if (typeof(T).IsEnum)
    {
        // TODO
        // Note: ReflectionExtensions.GetEnumMappings can be used as an example
    }

    return Equals(value, null) ? url : AddParameterInternal(url, parameterName, value.ToString());
}
```